addons/web/static/src/core/domain_selector/domain_selector.js

```diff
@@ -76,6 +76,9 @@ export class DomainSelector extends Component {
         }
         if (!defaultLeafDomain) {
             defaultLeafDomain = new Domain(await this.getDefaultLeafDomain(p.resModel));
+            if (status(this) === "destroyed") {
```

Comment:
Problem with owl: destruction is done after the check in protect and before here.
Waiting for a fix in owl.

Reply:
Fixed by https://github.com/odoo/owl/commit/7538aeae0ef1aa227e975a06965030135fc6b7b2.
We can merge now